games/r.yaml

```diff
@@ -205,7 +208,8 @@
 
 - name: RollerCoaster Tycoon 2
   meta:
-    genre: [simulation]
+    genre: [Simulation, Strategy]
```

Comment:
RTS is a better descriptor here, I believe than just "Strategy" but to be honest i don't think every "simulation" game needs to be tagged as strategy as well. I'd prefer just "simulation" as it's the most clear genre as a theme park simulator.